daml-assistant/daml-helper/src/DA/Daml/Helper/Start.hs

```diff
@@ -118,7 +118,7 @@ withSandbox :: StartOptions -> FilePath -> [String] -> [String] -> (Process () (
 withSandbox StartOptions{..} darPath scenarioArgs sandboxArgs kont =
     case sandboxChoice of
       SandboxClassic -> oldSandbox "sandbox-classic"
-      SandboxKV -> oldSandbox "sandbox"
+      SandboxKV -> oldSandbox "sandbox-kv"
```

Comment:
(There was some extra logic to deal with different platform versions here, but I dropped it since we're going to drop platform-version for 2.0.)